deepspeed/module_inject/replace_module.py

```diff
@@ -350,6 +372,9 @@ def set_lm_head(module):
             pbar.update(1)
             gc.collect()
         replaced_module = set_lm_head(replaced_module)
+        # conv2d tp module replace
+        if 'Yuan' in str(replaced_module):
```

Comment:
Does it mean we apply conv sharding only for models we know there is conv layer?

Reply:
Yes. I added the comment for helping to understand this situation~